data/policies/extend.cas

```diff
@@ -0,0 +1,15 @@
+resource foo {
+	allow(bar, foo, file, getattr);
```

Comment:
Could you add something using `this` like: `allow(bar, foo, this, setattr);`?

Reply:
Did you mean to replace "foo" with "this" instead of "file"?  Because using "this" as the third argument would be an error.

I'll have a local fix changing "foo" to "this" where applicable, which I'll push once I've addressed the other comments.  I think that addresses the concern.  Let me know.